frontend/src/layouts/BaseLayout.vue

```diff
@@ -157,24 +157,30 @@
               dense
               icon="fas fa-cog"
             />
-            <span class="dark-toggle text-caption q-ml-md">Advanced mode {{ advancedMode ? 'on' : 'off' }}</span>
+            <span class="dark-toggle text-caption q-ml-md">{{$t('Base-Layout.advanced-mode')}} {{ advancedMode ? $t('Base-Layout.on') : $t('Base-Layout.off') }}</span>
             <span>
               <base-tooltip class="q-ml-sm" icon="fas fa-question-circle">
-                Enables advanced features such as private key export, additional recipient ID options, and event
-                scanning settings.
-                <span class="text-bold">Use with caution!</span>
+                {{$t('Base-Layout.advanced-mode-description')}}
+                <span class="text-bold">{{$t('Base-Layout.caution')}}</span>
               </base-tooltip>
             </span>
           </p>
+          <!-- Language selection -->
+          <p>
+            <select class="text-caption" v-model="$i18n.locale">
+              <option value='en-us'>English</option>
+              <option value='zh-cn'>中文</option>
+             </select>
+             <span class="text-caption q-ml-md">Language</span>
+          </p>
         </div>
 
         <!-- Column 2: Built by ScopeLift -->
         <div class="col-xs-12 col-sm-4 q-mt-lg">
-          <p class="dark-toggle spaced-letters">About</p>
-          <p class="dark-toggle text-caption">
-            Built by
+          <p class="dark-toggle spaced-letters">{{$t('Base-Layout.about')}}</p>
+          <i18n path="Base-Layout.built-by" tag="p" class="dark-toggle text-caption">
             <a href="https://www.scopelift.co/" target="_blank" class="hyperlink">ScopeLift</a>
-          </p>
+          </i18n>
```

Comment:
Part of the above-mentioned README docs should include when to use these `<i18n>` tags vs. standard HTML tags